contracts/token/ERC20/ERC20.sol

```diff
@@ -104,7 +104,7 @@ contract ERC20 is IERC20 {
     balances_[_from] = balances_[_from].sub(_value);
     balances_[_to] = balances_[_to].add(_value);
     allowed_[_from][msg.sender] = allowed_[_from][msg.sender].sub(_value);
-    emit Transfer(_from, _to, _value);
```

Comment:
We can't remove the `Transfer` event from here because it's required by ERC20. If anything we have to add a new event.